sdk_extensions/resources/src/main/java/io/opentelemetry/sdk/extensions/resources/ProcessResource.java

```diff
@@ -79,4 +81,9 @@ protected Attributes getAttributes() {
 
     return attributes.build();
   }
+
+  @Override
+  public String getName() {
```

Comment:
Instead of adding `getName` which might have conflicting values and is a bit annoying to implement in resource providers, let's just match `getClass().getName()` vs the property